tests/containers/podman_ipv6.pm

```diff
@@ -21,7 +22,7 @@ sub _cleanup {
     $podman->cleanup_system_host();
 
     my $registry_ipv4 = script_output('dig +short registry.opensuse.org A | grep -v suse');
-    assert_script_run("iptables -D OUTPUT -d $registry_ipv4 -j DROP");
+    script_run("iptables -D OUTPUT -d $registry_ipv4 -j DROP");
```

Comment:
which means if something goes wrong and cleanup fail we will not notice :thinking:  can you elaborate why you decided to remove failure here ?

Reply:
In general you should test in `run()` not in `post_run_hook()`. But I'll revert this.